libraries/AP_Torqeedo/AP_Torqeedo_Backend_Serial.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+   This program is free software: you can redistribute it and/or modify
```

Comment:
At this point it seems like unnecessary complexity to have this AP_Troqeedo_Backend_Serial although I guess it makes sense if we're going to add ePropulsion support (which also uses serial).  The Torqeedo CAN protocol would not re-use this backend though... I guess I'm OK with this _serial backend staying although I'd probably lean towards removing it and keeping things a little simpler.